collectors/auth0/test/auth0_test.js

```diff
@@ -154,6 +154,48 @@ describe('Unit Tests', function () {
                 done();
             });
         });
+
+        it('log format when type is null or undefined', function (done) {
+            let ctx = {
+                invokedFunctionArn: auth0Mock.FUNCTION_ARN,
+                fail: function (error) {
+                    assert.fail(error);
+                    done();
+                },
+                succeed: function () {
+                    done();
+                }
+            };
+
+            Auth0Collector.load().then(function (creds) {
+                var collector = new Auth0Collector(ctx, creds);
+                auth0Mock.AUTH0_LOG_EVENT.type = null;
+                let fmt = collector.pawsFormatLog(auth0Mock.AUTH0_LOG_EVENT.type);
+                assert.equal(fmt.messageTypeId, undefined);
+                done();
+            });
+        });
+
+        it('log format when date is null or undefined', function (done) {
+            let ctx = {
+                invokedFunctionArn: auth0Mock.FUNCTION_ARN,
+                fail: function (error) {
+                    assert.fail(error);
+                    done();
+                },
+                succeed: function () {
+                    done();
+                }
+            };
+
+            Auth0Collector.load().then(function (creds) {
+                var collector = new Auth0Collector(ctx, creds);
+                auth0Mock.AUTH0_LOG_EVENT.date=null;
+                let fmt = collector.pawsFormatLog(auth0Mock.AUTH0_LOG_EVENT.date);
```

Comment:
just pass auth0Mock.AUTH0_LOG_EVENT not auth0Mock.AUTH0_LOG_EVENT.date

Reply:
I have updated this

Reply:
I still see auth0Mock.AUTH0_LOG_EVENT.date is passed as parameter to pawsFormatLog

Reply:
Actually, this is showing outdated and I have already made changes for this. You can check at
Pull Request Tab -> Auth0 Test Case Added -> Files changed -> collectors/auth0/test/auth0_test.js 

When type is null
<img width="1534" alt="image" src="https://user-images.githubusercontent.com/98916725/219572301-11fc693f-1c31-46bd-850b-c78d17345497.png">


When date is null
<img width="1536" alt="image" src="https://user-images.githubusercontent.com/98916725/219572491-1d550d85-2d86-4d13-b494-ce39c522113d.png">

Reply:
it was updated an hour ago didn't checked thanks